python/vyos/config_mgmt.py

```diff
@@ -0,0 +1,664 @@
+# Copyright 2023 VyOS maintainers and contributors <maintainers@vyos.io>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this library.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import re
+import sys
+import pwd
+import gzip
+import logging
+from typing import Optional, Tuple, Union
+from argparse import ArgumentParser, REMAINDER
+from shutil import copy
+from filecmp import cmp
+from time import time
+from datetime import datetime
+from difflib import unified_diff
+from tabulate import tabulate
+
+from vyos.config import Config
+from vyos.configtree import ConfigTree
+from vyos.remote import upload
```

Comment:
Since it's only used in one place, maybe make it a local import there?

Reply:
upload is now a local import